tsconfig.json

```diff
@@ -15,11 +15,7 @@
     "declaration": true,
     "skipLibCheck": true,
     "lib": ["es6"],
-    "types": [
-        "@types/node",
-        "@types/ws",
-        "eventemitter3"
-    ]
+    "typeRoots": ["node_modules/@types"]
```

Comment:
Not 100% sure. I was using the same general syntax as in the graphql-subscriptions project.